### ISSUE TITLE
Add support for PHP

### ISSUE THREAD

**Issue by standingdreams** [windwp/vscode-expand-region#11]
This extension doesn't work properly in PHP. If you attempt to expand the selection inside of a tag, it will either select the first line of the tag or, if the tag has an attribute, it will select the attribute.

**Comment by Grafikart** [comment]
Could you provide an example ? On my editor it expands this way : 

```
<p><?php echo $hello . "Test"; ?></p>
                          |
                        <--->
    <--------------------------->
   <----------------------------->
```

**Comment by standingdreams** [comment]
Fixed with #12

**Comment by standingdreams** [comment]
#12 Has fixed this issue
